server/tests/employer.form.test.js

```diff
@@ -150,4 +171,35 @@ describe('Server V1 Form Endpoints', () => {
       .send({ ...form, doesCertify: false });
     expect(res.statusCode).toEqual(400);
   });
+
+  it('Create new site, receive success', async () => {
+    const res = await saveSites(site);
+    const expectedRes = [
+      { siteId: 67, status: 'Success' },
+    ];
+    expect(res).toEqual(expectedRes);
+  });
+
+  it('Get sites, receive all successfully', async () => {
```

Comment:
I'm not super familiar with this testing framework, do the tests always run in order?

Reply:
Yes, we use `--runInBand` when running the test command